Upstream-Status: Backport

Signed-off-by: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
---
From 53189bba54cc9bb057f9e35ede472088bed73a1d Mon Sep 17 00:00:00 2001
From: mbjorge <mbjorge@chromium.org>
Date: Mon, 21 Nov 2016 18:39:22 -0800
Subject: [PATCH] [Buildfix] Fix enable_webrtc=false build.

BUG=667527
TEST=ninja content/renderer

Review-Url: https://codereview.chromium.org/2522663003
Cr-Commit-Position: refs/heads/master@{#433753}
---
 content/renderer/pepper/pepper_media_device_manager.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/content/renderer/pepper/pepper_media_device_manager.cc b/content/renderer/pepper/pepper_media_device_manager.cc
index 412c53f..5da6cb4 100644
--- a/content/renderer/pepper/pepper_media_device_manager.cc
+++ b/content/renderer/pepper/pepper_media_device_manager.cc
@@ -89,9 +89,9 @@ void PepperMediaDeviceManager::EnumerateDevices(
                  callback, ToMediaDeviceType(type)));
 #else
   base::ThreadTaskRunnerHandle::Get()->PostTask(
-      FROM_HERE,
-      base::Bind(&PepperMediaDeviceManager::DevicesEnumerated, AsWeakPtr(),
-                 callback, ToMediaDeviceType(type), MediaDeviceInfoArray()));
+      FROM_HERE, base::Bind(&PepperMediaDeviceManager::DevicesEnumerated,
+                            AsWeakPtr(), callback, ToMediaDeviceType(type),
+                            std::vector<MediaDeviceInfoArray>()));
 #endif
 }
 
-- 
2.9.3

